node/builder.go

```diff
@@ -163,6 +165,15 @@ func (nc *Builder) build(ctx context.Context) (*Node, error) {
 		return nil, err
 	}
 
+	repoPath, err := nc.Repo.Path()
+	if err != nil {
+		return nil, err
+	}
+
+	jBuilder, err := journal.NewZapJournalBuilder(fmt.Sprintf("%s/%s", repoPath, "journal.json"))
```

Comment:
The repo should be in control of its internals. Ask the repo to construct the path.